### PR TITLE
feat: v36 typography

### PR DESCRIPTION
**Pull request by limpbrains**
# Description

- Changing app font to *Inter Tight*
- removing all old fonts
- removing all icon fonts
- theme no longer depends on language

---

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Tests
- [ ] Detox test
- [ ] Unit test
- [x] No test

## QA Notes

Make sure that font looks fine on iOS and Android

### CODE REVIEW

File: src/screens/Widgets/WidgetFeedEdit.tsx [hunk 1]
```diff
@@ -219,9 +225,7 @@ export const WidgetFeedEdit = ({
 					</View>
 
 					{config.description && (
-						<Text01S style={styles.description} color="gray1">
-							{config.description}
-						</Text01S>
+						<Title color="gray1">{config.description}</Title>
```
Comment (pwltr) [inline src/screens/Widgets/WidgetFeedEdit.tsx]: Description should have font-weight "normal"

**Review by pwltr — CHANGES_REQUESTED** [review]

**Comment by pwltr** [comment]
Line-height for card text seems slightly off on iOS (left). Font seems good everywhere else.

Screenshot 2023-04-12 at 11 55 11

File: src/screens/Widgets/WidgetFeedEdit.tsx (hunk 1 above)
Comment (limpbrains) [reply src/screens/Widgets/WidgetFeedEdit.tsx]: fixed

**Comment by limpbrains** [comment]
> Line-height for card text seems slightly off on iOS (left). Font seems good everywhere else.
> 
> Screenshot 2023-04-12 at 11 55 11

I've noticed, that card size has been changed, now its 152 x 152. 
After I fixed this, now everything looks good enought ( iOS on the left ) 

image

**Review by pwltr — APPROVED** [review]

**Pull request merged**